src/en/news/crimson/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: content-simple
+title: Crimson Project
+order: 4
+---
+
+# {{ title }}
+
+## Goal
+
+Crimson is designed to be a faster OSD, in the sense that
+
+- It targets fast storage devices, like NVMe storage, to take advantage of the high performance of random I/O and high throughput of the new hardware.
+- It will be able to bypass the kernel when talking to networking/storage devices which support polling.
+- It will be more computationally efficient, so the load is more balanced in a multi-core system, and each core will be able to serve the same amount of IO with lower CPU usage.
+- It will achieve these goals by a design from scratch. It also utilizes modern techniques, like SPDK, DPDK and the Seastar framework, so it can 1) bypass the kernel, 2) avoid memcpy between, 3) avoid lock contention.
+
+Crismon will be a drop-in-replacement of the classic OSD. Our long-term goal is to redesign the object storage backend from scratch, This new backend is dubbed “Seastore”. The disk layout we will use for Seastore won’t necessarily be optimal for HDDs. But Seastore is still in its very early stage, and is still missing a detailed design -- we don’t have enough resources to work on it yet. As an intermediate solution, we are working on adapting bluestore to crimson-osd in the hope that we can use it for testing f he mid-term. But if the new object storage backend does not work well with HDD, we will continue using bluestore in crimson just for supporting HDD.
+
+We are using a C++ framework named Seastar to implement Crimson, but we are not porting classic OSD to Seastar, as the design philosophy and various considerations could be very different between them. So, again, please bear in mind, to port classic OSD to Seastar and adapt it to bluestore is never our final goal.
+
+The new Crimson OSD will be compatible with classic OSD, so existing users will be able to upgrade from classic OSD to Crimson OSD. Put in other words, crimson-osd will:
+
+- Support librados protocol
+  - Be able to talk to existing librados clients
+  - Be able to join an existing Ceph cluster as an OSD
```

Comment:
It sounds like Crimson OSD will be able to co-exist and talk with classic OSDs in the same cluster?

I'm not aware of any design/discussions around the mixed mode.